docs/reference/issuers/acme/http01.rst

```diff
@@ -26,6 +26,15 @@ using Ingress resources
        http01: {}
 
 
+To define which Kubernetes service type to use during challenge response specify the following http01 config
+
+.. code-block:: yaml
+       http01:
+         # Valid values are ClusterIP and NodePort
+         solverServiceType: ClusterIP
+
+By default type NodePort will be used when you don't set http01 or when you set solverServiceType to an empty string. This may change in future.
```

Comment:
I think we should heavily emphasis the cases in which you may want to change this, as it is *extremely* rare for users to ever need to touch this option.

Could you expand this part out to explain why you may want to change it, and also add a note to say that you probably *don't* want to change this?